src/app/workflow/workflow-detail/workflow-detail.component.ts

```diff
@@ -391,6 +349,114 @@ export class WorkflowDetailComponent implements OnInit, OnDestroy {
       });
   }
 
+
+  openCopy(content, jobId: string) {
+    this.getJob(content, jobId);
+  }
+
+  getJob(content, jobId: string ) {
+    this.jobService.getJob(jobId).subscribe(job => {
+        this.jobToCopy = job;
+        this.jobService.getPlugin(job.wippExecutable).subscribe(plugin => {
+          this.selectedSchema = this.pluginList.find(x => x.id === plugin.id);
+          this.jobModel['taskName'] = this.jobToCopy.name + '-copy';
+          this.jobModel['inputs'] = {};
+          const requests = [];
+          for (const input of Object.keys(this.jobToCopy.parameters)) {
+            if ((this.selectedSchema.properties.inputs.properties[input]['widget'] === 'search' || this.selectedSchema.properties.inputs.properties[input]['widget']['id'] === 'search') && this.jobToCopy.parameters[input].indexOf('{') === -1) {
+              const id = this.jobToCopy.parameters[input];
+
+              // Resolve AbstractFactory
+              const injectable = dataMap.get(this.selectedSchema.properties.inputs.properties[input]['format']);
+              // Inject service
+              this.service = this.injector.get(injectable);
+              requests.push(this.service.getById(id).pipe(map(response => {
+                  response['data'] = response;
+                  response['inputName'] = input;
+                  return response;
+                })
+              ));
+
+            }  else if (this.jobToCopy.parameters[input].indexOf('{') !== -1) {
```

Comment:
It seems that there should be a first "if/else widget is search" and then inside of the if "if/else this.jobToCopy.parameters[input].indexOf('{') === -1)", because the `{` can be found in regular string params as well (for example, stitching pattern)